packages/oc-template-handlebars-compiler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "oc-template-handlebars-compiler",
+  "version": "1.0.0",
+  "description": "OC-Template-Handlebars-Compiler",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/opencomponents/oc-template-handlebars.git"
+  },
+  "keywords": [],
+  "author": "Nick Balestra <nick@balestra.ch>",
```

Comment:
maybe me but personally I prefer much more to use a `person` object

https://docs.npmjs.com/files/package.json#people-fields-author-contributors

this is really just FYI